src/client/error.ts

```diff
@@ -12,12 +12,18 @@ export class QstashError extends Error {
 
 export class QstashRatelimitError extends QstashError {
   constructor(args: RateLimit) {
-    super(`You have been ratelimited. ${JSON.stringify(args)} `);
+    super(`Exceeded burst rate limit. ${JSON.stringify(args)} `);
```

Comment:
Btw, not exactly scope of this PR but parsing this error on users end is extremely difficult since error details are also stringified. Maybe we can address this issue too.

Reply:
do you mean we should add args as proper fields to the errors like: `QstashRatelimitError.limit`, `QstashRatelimitError.reset`?

Reply:
Exactly.